aopy/analysis.py

```diff
@@ -856,3 +854,146 @@ def linear_fit_analysis2D(xdata, ydata, weights=None, fit_intercept=True):
     linear_fit = reg_fit.coef_[0][0]*xdata.flatten() + reg_fit.intercept_
 
     return linear_fit, linear_fit_score, pcc_all[0], pcc_all[1], reg_fit
+
+def calc_activity_onset_accLLR(data, cond1, cond2, modality, bin_width, thresh_proportion=0.15, trial_average=True):
```

Comment:
Kind of nit-picky but i think the 'trial_average' input arg should be False. At least thats how I plan to use it.